Ical.Net/DataTypes/CalDateTime.cs

```diff
@@ -23,7 +23,7 @@ namespace Ical.Net.DataTypes;
 /// This is because RFC 5545, Section 3.3.5, does not allow for fractional seconds.
 /// </remarks>
 /// </summary>
-public sealed class CalDateTime : EncodableDataType, IComparable<CalDateTime>, IFormattable
+public sealed class CalDateTime : IComparable<CalDateTime>, IFormattable
```

Comment:
... or even struct?

Reply:
Yes, struct would certainly be good. But we'd have to deal with all the nullability stuff first, which is not perfectly straight forward. Next step could then be to do all that also for `Period`, `PeriodList` and maybe others.